AOloopControl/examples/scexao-vispyr-bin2-conf/simLHS/respM.fits.get.bash

```diff
@@ -6,9 +6,9 @@ if [ ! -f $FILENAME ]; then
 
     ggID='1MnNo9WfRHuKik3gpEHEjZHgt8_PM0T1a'
     ggURL='https://drive.google.com/uc?export=download'
-    filename="$(curl -sc /tmp/gcokie "${ggURL}&id=${ggID}" | grep -o '="uc-name.*</span>' | sed 's/.*">//;s/<.a> .*//')"
-    getcode="$(awk '/_warning_/ {print $NF}' /tmp/gcokie)"
-    curl -Lb /tmp/gcokie "${ggURL}&confirm=${getcode}&id=${ggID}" -o "${filename}"
+    filename="$(curl -sc /tmp/gcookie "${ggURL}&id=${ggID}" | grep -o '="uc-name.*</span>' | sed 's/.*">//;s/<.a> .*//')"
+    getcode="$(awk '/_warning_/ {print $NF}' /tmp/gcookie)" || touch /tmp/gcookie
+    curl -Lb /tmp/gcookie "${ggURL}&confirm=${getcode}&id=${ggID}" -o "${filename}"
```

Comment:
```&confirm=t``` is the URL option triggered by the **[Download anyway]** button when one navigates to to the URL, i.e. [https://drive.google.com/uc?export=download&id=1MnNo9WfRHuKik3gpEHEjZHgt8_PM0T1a](https://drive.google.com/uc?export=download&id=1MnNo9WfRHuKik3gpEHEjZHgt8_PM0T1a) from line 9 (see image below), so we should simply add that, or parse the URL from the [Download anyway] button.

![image](https://github.com/cacao-org/cacao/assets/1558679/f7cf0c1d-6842-4ec8-b3db-48fb9b82e7de)